.github/workflows/integration.yml

```diff
@@ -32,10 +32,16 @@ jobs:
       run: |
           brew install libomp
           echo 'export DYLD_LIBRARY_PATH=$(brew --prefix libomp)/lib:$DYLD_LIBRARY_PATH' >> $GITHUB_ENV
-    - name: Install dependencies
+    - if: matrix.python-version != 3.13
+      name: Install dependencies
       run: |
           python -m pip install --upgrade pip
           python -m pip install invoke .[pomegranate,xgboost,test]
+    - if: matrix.python-version == 3.13
+      name: Install dependencies
+      run: |
+          python -m pip install --upgrade pip
+          python -m pip install invoke .[xgboost,test]
```

Comment:
should we change this to support pomegranate again?

Reply:
@amontanez24 the version of pomegranate we are currently using is incompatible with 3.13, we'd need to update to the new pomegranate API first

Reply:
Should we update this message to say we don't support python 3.13 yet?
https://github.com/sdv-dev/SDMetrics/blob/7261608c5d7d24e3e2acd1c3126dabbdaa339867/sdmetrics/single_table/bayesian_network.py#L23

Reply:
@amontanez24 added